packages/graphback-datasync/src/providers/DatasyncMongoDBDataProvider.ts

```diff
@@ -97,51 +106,18 @@ export class DataSyncMongoDBDataProvider<Type = any> extends MongoDBDataProvider
       filter = {};
     }
 
-    filter._deleted = { ne: true };
+    filter[DataSyncFieldNames.deleted] = { ne: true };
 
     return super.count(filter);
   }
 
-  public sync(lastSync: string, context: GraphbackContext, filter?: any): Promise<Type[]> {
+  public sync(lastSync: number, context: GraphbackContext, filter?: any): Promise<Type[]> {
```

Comment:
`lastSync` is not a number, it is a `Date`. The scalar value is parsed as a Date.

```suggestion
  public sync(lastSync: Date, context: GraphbackContext, filter?: any): Promise<Type[]> {
```

Reply:
Good catch!

Reply:
Thanks! Addressed this in last push